filament/backend/include/backend/ShaderStageFlags.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2022 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef TNT_FILAMENT_BACKEND_SHADERSTAGEFLAGS_H
+#define TNT_FILAMENT_BACKEND_SHADERSTAGEFLAGS_H
+
+#include <backend/DriverEnums.h>
+
+#include <cstdint>
```

Comment:
use `stdint.h` (I know, I know, it's just to be consistant with the rest of the codebase)

Reply:
Ok :)